tests/runci/targets/Flash.hx

```diff
@@ -64,7 +64,7 @@ class Flash {
 					File.saveContent(mmcfgPath, "ErrorReportingEnable=1\nTraceOutputFileEnable=1");
 				}
 				switch (ci) {
-					case AzurePipelines:
+					case AzurePipelines, GithubActions:
```

Comment:
Let's use `AzurePipelines | GithubActions` in modern Haxe style \o/

Reply:
Hey, i thought `comma` is added after to have less overlap with `bitwise OR` in switchs ;)